modules/osgi-karaf/src/main/resources/features.xml

```diff
@@ -21,9 +21,9 @@
     <feature name="ignite-all" version="${project.version}" description="Apache Ignite :: All">
         <details>
             <![CDATA[Aggregate feature for installing all Apache Ignite module + their dependencies.
-            
+
```

Comment:
Can you please revert all changes in this file other than the relevant one, avoid showing large diff?

Reply:
Sure.

Reply:
It's OK, I'll do the rest before merge to master